### ISSUE TITLE
Weight voting timeout should be set to 1 week after the CTF scoreboard has been made available

### ISSUE THREAD

**Issue by overfl0** [ctftime/ctftime.org#43]
I've just taken part in UCSB iCTF 2016 but I cannot vote yet because the scoreboard is not available and CTFtime cannot verify that I've actually taken part in that CTF.
Additionally, if the organizers don't publish the results until the 11th of March, I won't be able to vote at all because the time to vote will have ended by then.

As of now, from what I can see, only people in the top50 for 2016 are able to vote for such CTFs.

Proposition: extend the voting time to one week after the results are published.
If the results are not published for a certain time (one month?), set the weight to 0 (to prevent bad CTF organizers withholding the results for ~6 months and then voting themselves in the voting time window,  when everyone else has forgotten about the competition).

**Comment by Pharisaeus** [comment]
I agree. Additionally this should also apply if the "max weight" was modified for some reason - the voting time should be extended from this point. Otherwise we might get curious cases like https://ctftime.org/event/400 where max weight was initially set to 25, and so people voted, and then it was changed to 80, but almost no-one had noticed this/had time to change their vote.

**Comment by kyprizel** [comment]
It's hard to track if voting process run or not. So - no scorebord from orgs within a week = no voting.

**Comment by Pharisaeus** [comment]
@kyprizel can't you just enable voting only after the scorboard gets posted?

As for HITCON, my point was just that initially, after the event ended, you could vote for max 25p, and so the teams voted "max". And then somewhere at the end of voting window the max was raised to 80 and most teams had no chance of changing the vote. So in the end, even though people voted "max", it actually lowered the final rating.
